clang/test/CAS/fdepscan-result-cache-output-path.c

```diff
@@ -0,0 +1,15 @@
+// TODO: Test should be updated to not depend on a working cache at default location, which is out of test/build directory.
+//
+/// Test absolute path output.
+// RUN: rm -rf %t && mkdir %t
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=inline -c -x c %s -o %t/out.o
+// RUN: ls %t/out.o && rm %t/out.o
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=inline -c -x c %s -o %t/out.o
+// RUN: ls %t/out.o
+//
+/// Test relative path output.
+// RUN: cd %t
+// RUN: %clang -target x86_64-apple-macos11 -fdepscan=inline -c -x c %s -o out.o
+// RUN: ls out.o && rm out.o
```

Comment:
If the input is canonicalized, we should be able to get a cache hit here, since we've already done this with an absolute path.